README.md

```diff
@@ -2,110 +2,183 @@
-
-### import categories
-1. Use the category structure as in categories.csv
-
-2. Import it with https://impex.sphere.io/commands/category-import
-
-### import products
-1. Use the product structure as in products.csv
-2. Import it with https://impex.sphere.io/commands/product-import
-
 
+## How to create a project with Sunrise data
+
+Before starting the import, make sure you have access to the [Admin Center](https://admin.commercetools.com) and the [IMPEX tool](https://impex.commercetools.com/). You will also need to run the application in this repository, check that you meet the [requirements](#requirements).
+
+### 1. Set up your project
+1. Open the [Admin Center](https://admin.commercetools.com) and create an empty project (without sample data).
+2. Select your new project and go to **`Settings`**:
+    - In the tab **`International`**:
+        - **Currencies**: Euro (EUR)
+        - **Countries**: Germany (DE), Austria (AT)
+        - **Languages**: German (DE), English (EN)
+        - **Zone**: Europe with DE, AT
+    - In the tab **`Taxes`**:
+        - **Tax Category name**: "standard"
+        - **Tax rates**:
+            - Germany: 19% incl. in price
+            - Austria: 20% incl. in price
+    - In the tab **`Shipping Methods`**:
+        - **Standard shipping**:
+            - Name: "Standard"
+            - Description: "Delivery in 5-6 working days"
+            - Price: 3 EUR
+            - Free above: 200 EUR
+            - Set as default: true
+        - **Express shipping**:
+            - Name: "Express"
+            - Description: "Delivery the same day"
+            - Price: 10 EUR
+    - In the tab **`Customer Groups`**:
+        - **Name**: "b2b"
+        
+### 2. Import channels
+1. [Clone](https://help.github.com/articles/cloning-a-repository/) this repository to your computer.
+2. Go to the root folder of the cloned project and create a file named `payload-channels.json` with the following content:
+    ```json
+    {
+      "commercetools": {
+        "projectKey": "your-project-key",
+        "clientId": "your-client-id",
+        "clientSecret": "your-client-secret",
+        "authUrl": "https://auth.sphere.io",
+        "apiUrl": "https://api.sphere.io"
+      },
+      "jobs": [
+        {
+          "name": "importJoyrideChannelsJob",
+          "channelsResource": "https://raw.githubusercontent.com/commercetools/commercetools-sunrise-data/master/data/channels/channels.json",
+          "typesResource": "https://raw.githubusercontent.com/commercetools/commercetools-sunrise-data/master/data/channels/types.json"
+        }
+      ]
+    }
+    ```
+3. Adapt the file with your commercetools project credentials.
+4. Run the following command:
+    ```bash
+    export PAYLOAD_FILE=payload-channels.json && mvn spring-boot:run -Dstart-class=com.commercetools.dataimport.all.PayloadJobMain
+    ```
+
+### 3. Import catalog
```

Comment:
why not use the importApp for all of it?    That's much easier and people have to set up the maven stuff anyways.

Reply:
Because we want to remove the import app eventually, and then use other applications that are better suited for it: node.js scripts for example. This import app served its purpose at the beginning, but we cannot maintain it and even less continue extending it. Currently we are just duplicating code with other client teams.

Reply:
right, that's what we had discussed anyways.   

let's try a script that automates the other CLIs pretty directly now, it's good input and testing ground for the CLI team, too.   I can give it a try the next time I am doing a SUNRISE setup if nobody else tried it before.